components/dashboard/src/prebuilds/list/ConfigurationInput.tsx

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright (c) 2024 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { FC, useCallback, useMemo, useState } from "react";
+import { useConfiguration, useListConfigurations } from "../../data/configurations/configuration-queries";
+import { Combobox, ComboboxElement, ComboboxSelectedItem } from "@podkit/combobox/Combobox";
+
+interface RepositoryFinderProps {
+    selectedConfigurationId?: string;
+    disabled?: boolean;
+    expanded?: boolean;
+    onChange: (configurationId: string) => void;
+}
+
+export default function ConfigurationDropdown({
+    selectedConfigurationId,
+    disabled,
+    expanded,
+    onChange,
+}: RepositoryFinderProps) {
+    const [searchTerm, setSearchTerm] = useState("");
+    const { data, isLoading } = useListConfigurations({ searchTerm, sortBy: "creationTime", sortOrder: "desc" });
+    const configurations = data?.pages[0].configurations;
```

Comment:
Should we use another query if we only care about first page?

And if it's Gitpod org on cloud, if I want to search Gitpod configuraiton, maybe it's impossible -> almost every configurations' context is `https://github.com/gitpod-io/xxx`. Should we have pagniation works and allow Inf scroll?

Seems too much for this PR, can be follow up or found other solution

Reply:
We have the same problem on workspace create that if configuration names are the same, it's hard to identify which to choose.

Maybe add a link/button in Configuration to allow it to link to prebuilds of current configuration

PS. should we align styles for dropdown (dropdown options / dropdown input)? 
|Repository|Status|
|:-:|:-:|
|<img width="561" alt="image" src="https://github.com/gitpod-io/gitpod/assets/20944364/68cc886a-409a-4d31-98b7-d0f8ae0948f6">|<img width="508" alt="image" src="https://github.com/gitpod-io/gitpod/assets/20944364/df45f1b8-f656-4a6a-afa6-84fff77769ba">|

Reply:
Yikes, you're right, thanks for pointing this out. Let's indeed holf off on this until a follow-up (I would want to redo the whole selector again and do it properly).

For the style alignment, let's also follow-up on this. This works and although it's ugly, it's a pain to style right now as it uses `Combobox` under the hood which has many style assumptions.